### PR TITLE
Fix Debounce and ProgressBar animate

### PR DESCRIPTION
**Pull request by henmerlin**

### CODE REVIEW

File: acs2/test/tests/junit/consulta/api/GetSupported.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+        try {
+            EquipamentoDAO d = new EquipamentoDAO();
+            NbiDeviceData eqp;
+
+            d.getSup();
+
+            assertTrue(true);
+        } catch (Exception ex) {
+            assertTrue(false);
```
Comment (duchuyctlk) [inline acs2/test/tests/junit/consulta/api/GetSupported.java]: How about `fail();`?

File: acs2/web/resources/vue-components/template.js [hunk 2]
```diff
@@ -99,27 +103,23 @@ var bar = new ProgressBar.Line('#bar', {
...
-        i++
-        if (i < 10) {
-            bar.animate(i / 10);
-        } 
-    }, val)
+    bar.animate(0.96);
 }
 
+function loadingBarLong(val) {
+    if (val) {
```
Comment (duchuyctlk) [inline acs2/web/resources/vue-components/template.js]: Could this `if-else` statement be simplified to `bar.animate(0.98, {duration: val ? val : 12000});` ?

Comment (Agibert) [inline acs2/web/resources/vue-components/template.js]: Yes, thanks for the tip, saves some code lines

File: acs2/test/tests/junit/consulta/api/GetSupported.java (hunk 1 above)
Comment (Agibert) [inline acs2/test/tests/junit/consulta/api/GetSupported.java]: Thank you for the tip but fail(); would finish the void and in the future there may be some coding after the assertTrue(false);.... We could have used fail(); for now but besides killing the void I don't know if there is any difference between these methods...

**Pull request merged**